demo/src/index.tsx

```diff
@@ -21,687 +21,134 @@ const renderFn = (expanded: boolean): JSX.Element => {
-                </div>
-                <AccordionItemPanel>
-                    <p>
-                        Everything mentioned in the installation process should
-                        already be done.
-                    </p>
-                    <p>
-                        # Make sure you use the right node version.
-                        <br />
-                        nvm use
-                        <br />
-                        # Start the server and the development tools.
-                        <br />
-                        yarn start-demo
-                    </p>
-                </AccordionItemPanel>
-            </AccordionItem>
-        </Accordion>
-
-        <h2 className="u-margin-top">A bit of animation on the arrow?</h2>
+        <p>TODO</p>
 
-        <Accordion>
-            <AccordionItem>
-                <AccordionItemHeading className="accordion__heading accordion__heading--animated">
-                    <h3 className="u-position-relative">
-                        Animated Accessible Accordion
-                        <div className="accordion__arrow" role="presentation" />
-                    </h3>
-                </AccordionItemHeading>
-                <AccordionItemPanel>
-                    <p>Did you notice the animation on the arrow?</p>
-                </AccordionItemPanel>
-            </AccordionItem>
-            <AccordionItem className="accordion__item">
-                <AccordionItemHeading className="accordion__heading accordion__heading--animated">
-                    <h3 className="u-position-relative">
-                        How to?
-                        <div className="accordion__arrow" role="presentation" />
-                    </h3>
-                </AccordionItemHeading>
-                <AccordionItemPanel>
-                    <p>Check css/demo.css in the demo/ folder :)</p>
-                </AccordionItemPanel>
-            </AccordionItem>
-        </Accordion>
-
-        <h2 className="u-margin-top">Only one item</h2>
+        <h2 className="u-margin-top">Informative onChange</h2>
 
-        <Accordion>
-            <AccordionItem>
-                <AccordionItemHeading>
-                    <h3 className="u-position-relative">
-                        Single item
-                        <div className="accordion__arrow" role="presentation" />
-                    </h3>
-                </AccordionItemHeading>
-                <AccordionItemPanel>
-                    <p>Why would you need more than one?</p>
-                </AccordionItemPanel>
-            </AccordionItem>
-        </Accordion>
+        <p>
+            When you use the <strong>onChange</strong> prop, you can get
+            feedback about which items are expanded.
+        </p>
 
-        <h2 className="u-margin-top">Informative onChange</h2>
+        <p>
+            In this example, we are simply logging the uuids of the expanded
+            items to the console.
```

Comment:
I didn't think it hurt to explicitly instruct users to have a look at the console (maybe could add a line after 'In this example, we are simply logging the uuids of the expanded items to the console' saying something like, 'Have a look to see this in action'). Just a minor thought.

Reply:
Done :)